kustomization/components/kube-vip/daemonset.yml

```diff
@@ -11,11 +17,15 @@ spec:
     metadata:
       labels:
         app.kubernetes.io/name: kube-vip-ds
+        app.kubernetes.io/version: v0.8.10
     spec:
       affinity:
         nodeAffinity:
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
+              - matchExpressions:
+                  - key: node-role.kubernetes.io/master
+                    operator: Exists
```

Comment:
I wonder why this is needed - k3s seems to mark the control plane nodes with both roles

Reply:
It's legacy.  They used to use this and moved to controlplane, but don't want to break stuff (yet).